### PR TITLE
ticdc: add usage for eventually consistent replication

### PR DESCRIPTION
**Pull request by amyangfei**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Add usage document for redo log based eventually consistent replication in TiCDC.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by ben1009 — APPROVED** [review]

File: ticdc/manage-ticdc.md [hunk 1]
```diff
@@ -762,3 +762,50 @@ cdc cli --pd="http://10.0.10.25:2379" changefeed query --changefeed-id=simple-re
...
+## 灾难场景的最终一致性复制
+
+从 v5.3.0 版本开始，TiCDC 开始提供灾难场景下的最终一致性复制能力。具体解决的场景是，当生产集群（即 TiCDC 同步的上游集群）发生灾难、且短时间内无法恢复对外提供服务，TiCDC 需要具备保证从集群数据一致性的能力，并允许业务快速的将流量切换至从集群，避免数据库长时间不可用而对业务造成影响。
+
+该功能支持 TiCDC 将 TiDB 集群的增量数据复制到备用关系型数据库 TiDB/Aurora/MySQL/MariaDB，在 TiCDC 正常同步没有延迟的情况下，上游发生灾难后，可以在 30 分钟内将下游集群恢复到上游的某个 snapshot 状态，并且允许丢失的数据小于 5 分钟。即 RPO <= 30min，RTO <= 5min。
+
+### 使用前提
+
+- 准备好具有高可用的 S3 存储或 NFS 系统，用于存储 TiCDC 的实时增量数据备份文件，在上游发生灾难情况下该文件存储可以访问。
+- TiCDC 对需要具备灾难场景最终一致性的 changefeed 开启该功能，新增配置在 changefeed 的配置文件中。
```
Comment (hfxsd) [inline ticdc/manage-ticdc.md:775]: 这句话看不懂。建议改写下。

**Review by hfxsd — APPROVED** [review]

**Review by Liuxiaozhen12 — APPROVED** [review]

File: ticdc/manage-ticdc.md [hunk 2]
```diff
@@ -762,3 +762,50 @@ cdc cli --pd="http://10.0.10.25:2379" changefeed query --changefeed-id=simple-re
...
+
+### 使用前提
+
+- 准备好具有高可用的 S3 存储或 NFS 系统，用于存储 TiCDC 的实时增量数据备份文件，在上游发生灾难情况下该文件存储可以访问。
+- TiCDC 对需要具备灾难场景最终一致性的 changefeed 开启该功能，开启方式是在 changefeed 配置文件中增加以下配置：
+
+```toml
+[consistent]
+# 一致性级别，选项有：
+# - none： 默认值，非灾难场景，提供 finished-ts 情况下的最终一致性。
```
Comment (hfxsd) [inline ticdc/manage-ticdc.md:780]:
```suggestion
# - none： 默认值，非灾难场景，只有在任务指定 finished-ts 情况下保证最终一致性。
```

**Pull request merged**